test/blackbox-tests/test-cases/configurator-import.t/dune

```diff
@@ -0,0 +1,12 @@
+(rule
+ (targets c_flags.sexp)
+ (deps
+  (:deps-conf configure/configure.exe))
+ (action
+  (run %{deps-conf})))
```

Comment:
This can be `(rule (with-stdout-to c_flags.sexp (run configure/configure.exe)))` if you change the configure script to write to stdout instead of to a file.